interpreter/golabels/runtime_data.go

```diff
@@ -0,0 +1,115 @@
+package golabels // import "go.opentelemetry.io/ebpf-profiler/interpreter/golabels"
+
+// #include <stdlib.h>
+// #include "../../support/ebpf/types.h"
+import "C"
+
+// defaultVersion is used if the go binary has an unrecognized major+minor version.
+// Consider bumping this whenever a new version of Go is released.
+var latestVersion = "go1.24"
+
+// Offsets come from DWARF debug information, use tools/gooffsets to extract them.
+// However since DWARF information can be stripped we record them here.
+// TODO: Should we look for DWARF information to support new versions
+// automatically when available?
+var allOffsets = map[string]C.GoCustomLabelsOffsets{
```

Comment:
Could this be a function and do `go/version` `Compare` to deduplicate the data. Seems almost all legacy versions use the same data. Doing this would also remove the need to deduce the "major" version earlier and make sure we can test icnluding minor versions.